test/e2e/domain_mapping_test.go

```diff
@@ -108,6 +108,16 @@ func domainList(r *test.KnRunResultCollector, domainName string) {
 }
 
 func domainDescribe(r *test.KnRunResultCollector, domainName string, tls bool) {
+	k := test.NewKubectl(r.KnTest().Kn().Namespace())
+	// Wait for Domain Mapping URL to be populated
+	for i := 0; i < 10; i++ {
+		out, err := k.Run("get", "domainmapping", domainName, "-o=jsonpath='{.status.url}'")
+		assert.NilError(r.T(), err)
+		if len(out) > 0 {
+			break
+		}
+		time.Sleep(time.Second)
+	}
```

Comment:
Checking the domain mapping content immediately after creation leads to false-positive URL missing fails. The following simple look should make sure URL is there to be checked by the assert later.